stream-chat-android/src/main/java/com/getstream/sdk/chat/viewmodel/messages/MessageListItemLiveData.kt

```diff
@@ -51,8 +51,7 @@ import java.util.Date
  */
 internal class MessageListItemLiveData(
     private val currentUser: User,
-    newMessagesLd: LiveData<List<Message>>,
-    oldMessagesLd: LiveData<List<Message>>,
```

Comment:
I like it, it seemed weird that we had two sources for actual and old messages

Reply:
It was used for "new messages counter", but it wasn't working properly, and at the end it wasn't needed at all